<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>entry_frame</class>
 <widget class="QFrame" name="entry_frame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>323</width>
    <height>462</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="2">
    <widget class="QPushButton" name="browse_button">
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="QPushButton" name="load_button">
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="value_entry"/>
   </item>
   <item row="2" column="0">
    <widget class="QComboBox" name="type_selector"/>
   </item>
   <item row="6" column="0" colspan="3">
    <widget class="QPushButton" name="finish_button">
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="3">
    <widget class="QPushButton" name="remove_button">
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QTreeView" name="list_view"/>
   </item>
   <item row="1" column="0">
    <widget class="QLineEdit" name="type_header"/>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="value_header"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
